pycsw/server.py

```diff
@@ -116,11 +116,10 @@ def __init__(self, rtconfig=None, env=None, version='3.0.0'):
                     with codecs.open(rtconfig, encoding='utf-8') as scp:
                         self.config.readfp(scp)
         except Exception as err:
-            LOGGER.exception('Could not load user configuration: %s', err)
+            msg = 'Could not load configuration'
+            LOGGER.exception('%s %s: %s', msg, rtconfig, err)
             self.response = self.iface.exceptionreport(
-                'NoApplicableCode', 'service',
-                'Error opening configuration %s' % rtconfig
-            )
+                'NoApplicableCode', 'service', msg)
```

Comment:
Yes this is nice!

I also like the fact that we're not giving away the `rtconfig` value in the exception report.